pkgs/top-level/all-packages.nix

```diff
@@ -24914,9 +24914,9 @@ with pkgs;
 
   sdrplay = callPackage ../applications/radio/sdrplay { };
 
-  sdrpp = pin-to-gcc12-if-gcc13 (callPackage ../applications/radio/sdrpp {
+  sdrpp = callPackage ../applications/radio/sdrpp {
     inherit (darwin.apple_sdk.frameworks) AppKit;
-  });
+  };
```

Comment:
Maybe move it here:
```suggestion
    rtl-sdr = rtl-sdr-osmocom; # osmocom better w/ rtlsdr v4
  };
```

Reply:
I think moving that configuration there is probably better done in a separate change.